app/models/declaration/ProcedureCodesData.scala

```diff
@@ -24,6 +24,8 @@ case class ProcedureCodesData(procedureCode: Option[String], additionalProcedure
   def toProcedureCode(): ProcedureCodes = ProcedureCodes(procedureCode, None)
 
   def containsAdditionalCode(code: String): Boolean = additionalProcedureCodes.contains(code)
+
+  def summaryValue: String = procedureCode.fold("")(code => (Seq(code) ++ additionalProcedureCodes).mkString(" "))
```

Comment:
As above, I'd prefer we didn't clutter the model with helper methods that don't naturally belong there. If the summary page is the only place that needs it, lets just stick the code there.

*Unless* every model will have a `summaryValue` method, in which case lets create a `trait` for it

Reply:
Moved to the view

Reply:
Looks like it needs deleting in the model